icon.c

```diff
@@ -170,6 +170,13 @@ static char *resolve_icon(struct mako_notification *notif) {
 		for (size_t i = 0; i < found_count; ++i) {
 			char *relative_path = icon_glob.gl_pathv[i];
 
+			if (!strcmp(relative_path+strlen(relative_path)-4, ".svg")) {
```

Comment:
Maybe a better approach would be to check if the size subdir is named "scalable"?

Reply:
Would have to check for "scalable" and "symbolic", but yeah, possibly. There definitely seem to be themes that have more complex SVGs for bigger sizes, so not throwing away sizing information for them may be desirable.

On the other hand I don't know if other icon themes use different names for this; though spot checking the arch repos that does seem to be a common theme.

For my usecase this definitely works, if you want I can update this PR to this commit:
https://github.com/Tharre/mako/commit/f7eed4670584317078340d4a45611347afc14bb3

Reply:
Hm, right, I suppose that to do things the Right Way, we'd need to parse the theme .ini files... Although, maybe we should keep parsing the size if there are .svg files in a directory with a fixed size...